### PR TITLE
fix rendering of `muc_roomid_policy_hint`

### PR DESCRIPTION
**Pull request by ChaosKid42**
This PR fixes the rendering of `muc_roomid_policy_hint`.

### CODE REVIEW

File: src/templates/add_chatroom_modal.js [hunk 1]
```diff
@@ -32,7 +33,7 @@ export default (o) => html`
                         ${ (o.muc_roomid_policy_error_msg) ? html`<label class="roomid-policy-error">${o.muc_roomid_policy_error_msg}</label>` : '' }
                         <input type="text" required="required" name="chatroom" class="form-control roomjid-input" placeholder="${o.chatroom_placeholder}"/>
                     </div>
-                    ${ o.muc_roomid_policy_hint ?  html`<div class="form-group">{{o.muc_roomid_policy_hint}}</div>` : '' }
```
Comment (jcbrand) [inline src/templates/add_chatroom_modal.js:35]: This makes me nervous :)

Please use the XSS library to filter the policy hint for malicious scripts before passing it to `unsafeHTML`.

Comment (ChaosKid42) [reply src/templates/add_chatroom_modal.js:35]: It's already there: https://github.com/conversejs/converse.js/blob/master/src/converse-muc-views.js#L545

Comment (jcbrand) [reply src/templates/add_chatroom_modal.js:35]: ah ok, can you please move that to the template? 

I think it makes sense to have it all in one place, then one can be sure that the HTML is being escaped.

Comment (ChaosKid42) [reply src/templates/add_chatroom_modal.js:35]: Done

**Pull request merged**